src/components/lib/crt/crt.h

```diff
@@ -80,6 +90,12 @@ struct crt_comp_resources {
 	vaddr_t     info;
 };
 
+struct crt_chkpt {
+    struct crt_comp *c;
```

Comment:
you're mixing spaces and tabs here. See the mismatch on github to make it obvious.